python/src/spark_rapids_ml/classification.py

```diff
@@ -1163,6 +1163,16 @@ def _set_params(self, **kwargs: Any) -> "LogisticRegression":
             self._set_cuml_reg_params()
         return self
 
+    def copy(
```

Comment:
Is there a way to move this to base class in core and somehow leverage _set_params which already calls set_cuml_reg_params().  Or refactor _set_params a bit to enable this?

Reply:
Good suggestion. Revised to have this moved to base class in params.py.